rfcs/70-book_evolution_in_main.md

```diff
@@ -0,0 +1,40 @@
+# Feature Name: `book_evolution_in_main`
+
+## Summary
+
+Reduce the cost of starting the book, and of book evolutions with new versions of Bevy
+
+## Motivation
+
+I want the Bevy Book to get better faster. We've tried to make it evolve in bevy-website repository, but getting to a publishable state means getting the content, the CI and the formatting ready all at once and lead to a long lived `new-book` branch that is in limbo. The current approach would also mean that for a new version of Bevy, we would have to pull a new branch, make it use the next version of Bevy, update everything, then merge it back on the website for publication. I fear this will make the release process longer.
+
+I'm opening this as a RFC as it has moving parts across 2 repositories, and it's an evolution on [RFC #23: Quick Start Book](https://github.com/bevyengine/rfcs/blob/main/rfcs/23-quick_start_book.md).
+
+## User-facing explanation
+
+As a user wanting to contribute to the book, I can do so on the Bevy main repository.
+
+## Implementation strategy
+
+- Add a new folder in Bevy main repository for the book
+- Start accepting contributions for it, with the layout / rules from [RFC #23: Quick Start Book](https://github.com/bevyengine/rfcs/blob/main/rfcs/23-quick_start_book.md)
+- Set up main repository CI to check that the book content is up to date with the code
+  - For new book PRs, this should be blocking
```

Comment:
How are these PR types distinguished? If it modifies any code in that folder at all?

Reply:
if the build fail:
- if the /book folder has been changed, fail the CI
- else, put a tag on the PR (or a comment, or open an issue)